controllers/managedocs_controller_test.go

```diff
@@ -44,14 +45,24 @@ var _ = Describe("ManagedOCS controller", func() {
 	}
 
 	Context("reconcile()", func() {
-		When("there is no storage clsuter resource in the cluster", func() {
-			It("should create a new storage cluster", func() {
+		When("There is no add-on parameters secret in the cluster", func() {
```

Comment:
The string specified by `When` is concatenated into a full sentence in the test output as such it should always start with a lowercase letter.